tidb-resource-control.md

```diff
@@ -0,0 +1,171 @@
+
+<CustomContent platform="tidb-cloud">
+
+> **Note:**
+>
+> The parameter `resource-control.enabled` is disabled by default. It does not support dynamic modification. You need to contact the [PingCAP support team](/tidb-cloud/tidb-cloud-support.md) to enable it for TiDB Cloud Dedicated Tier clusters.
+
+</CustomContent>
+
+The results of the combinations of these two parameters are shown in the following table.
+
+| `resource-control.enabled`  | `tidb_enable_resource_control`= ON   | `tidb_enable_resource_control`= OFF  |
+|:----------------------------|:-------------------------------------|:-------------------------------------|
+| `resource-control.enabled`= true  |  Flow control and scheduling (recommended) | Invalid combination      |  
+| `resource-control.enabled`= false |  Only flow control (not recommended)                 | The feature is disabled. |
+
+## How to use resource control
+
+To create, modify, or delete a resource group, you need to have the `SUPER` or `RESOURCE_GROUP_ADMIN` privilege.
+
+You can create a resource group in the cluster by using [`CREATE RESOURCE GROUP`](/sql-statements/sql-statement-create-resource-group.md), and then bind the users to a specific resource group by using [`CREATE USER`](/sql-statements/ sql-statement-create-user.md) or [`ALTER USER`](/sql-statements/sql-statement-alter-user.md).
+
+For an existing resource group, you can modify the read and write quota of the resource group by using [`ALTER RESOURCE GROUP`](/sql-statements/sql-statement-alter-resource-group.md). The quota changes to the resource group take effect immediately.
+
+You can delete a resource group by using [`DROP RESOURCE GROUP`](/sql-statements/sql-statement-drop-resource-group.md).
+
+> **Note:**
+>
+> - When you bind a user to a resource group by using `CREATE USER` or `ALTER USER`, it will not take effect for the user's existing sessions, but only for the user's new sessions.
+> - If a user is not bound to a resource group or is bound to a `default` resource group, the user's requests are not subject to TiDB's flow control restrictions. The `default` resource group is currently not visible to the user and cannot be created or modified.
+
+### Step 1. Enable the resource control feature
+
+Enable the resource control feature.
+
+```sql
+SET GLOBAL tidb_enable_resource_control = 'ON';
+```
+
+<CustomContent platform="tidb">
+
+In TiKV, set the parameter [`resource-control.enabled`](/tikv-configuration-file.md#resource-control) to `true`.
+
+</CustomContent>
+
+<CustomContent platform="tidb-cloud">
+
+In TiKV, set the parameter [`resource-control.enabled`](https://docs.pingcap.com/tidb/stable/tikv-configuration-file#resource_control) to `true`. The parameter `resource-control.enabled` is disabled by default. You need to contact the [PingCAP support team](/tidb-cloud/tidb-cloud-support.md) to enable it.
```

Comment:
```suggestion
In TiDB Cloud, the value of the TiKV `resource-control.enabled` parameter is  `false` by default and does not support dynamic modification. If you need to enable it for TiDB Cloud Dedicated Tier clusters, contact [TiDB Cloud Support](/tidb-cloud/tidb-cloud-support.md).
```

Reply:
https://docs.pingcap.com/tidb/stable/tikv-configuration-file#resource_control won't work because stable is for v6.5. We can skip providing the link for TiDB Cloud because Cloud users cannot modify it and need to seek support if necessary.

Reply:
I think we can keep it because we might forget to add it back in the 7.x version.

Reply:
What's your opinion? @songrijie

Reply:
At this moment, we'd better be conservative to public Cloud customers. I would like say it's not yet certified in TiDB Cloud.

Reply:
/cc @easonn7